sdk/build.gradle

```diff
@@ -58,6 +65,9 @@ dependencies {
     androidTestImplementation 'org.hamcrest:hamcrest-library:1.3'
 
     debugImplementation 'androidx.fragment:fragment-testing:1.4.1'
+
+    compileOnly 'com.google.code.findbugs:annotations:3.0.1'
+    // compileOnly 'com.google.code.findbugs:jsr305:3.0.2'
```

Comment:
Should we remove or uncomment this?

Reply:
Good catch, will remove it

To share a bit more context: I have tried to "make lombok and spotbugs friends". `lombok.config` even has dedicated property to suppress NPE-related warnings on lombok-generated getters (`lombok.extern.findbugs.addSuppressFBWarnings = true`). Unfortunatelly it doesn't work out of box.

Because `spotbugs` doesn't consider those warnings as critical, I updated reportLevel to high https://github.com/hCaptcha/hcaptcha-android-sdk/pull/42/files#diff-af6fe13f9278bb4fd072ec1a44a4332f16cdff38b4756f6fa4f40bba9e87b2a2R194

Reply:
Done